pxtlib/hf2.ts

```
@@ -270,6 +282,16 @@ namespace pxt.HF2 {
             this.eventHandlers[id + ""] = f
         }
 
+        sendCustomEventAsync(type: string, payload: Uint8Array): Promise<void> {
+            if (type == CUSTOM_EV_JACDAC)
+                if (this.jacdacAvailable)
+                    return this.talkAsync(HF2_CMD_JDS_SEND, payload)
+                        .then(() => { })
```

we should swallow the error here